.github/workflows/blank.yml

```diff
@@ -0,0 +1,61 @@
+name: CI
+
+on:
+  # Triggers the workflow on pull request events but only for the main branch
+  pull_request:
+    branches: [ main ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a two jobs called "backend" and "frontend"
+  backend:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+    env: 
+      working-directory: src/API/
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+      - uses: actions/setup-dotnet@v1.7.2
+        with: 
+            dotnet-version: 5.0.100-preview.2.20176.6
```

Comment:
Hm, is it possible to have some basic dot.net version 5.0 without any preview?

Reply:
Changed to latest possible version.